### DISCUSSION TITLE
How do we sync together?

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/sig-release#3] [General]
Open discussion, how we align together to provide open source information?

**Comment by efficient0482** [comment]
# Suggestion to organize the Sync:
- create a rough schedule e.g. every 2 weeks
- create a discussion for the next scheduled meeting and collect topic within this discussion
- add a link to a online meeting, such that every interested person may join
## Addionaly
- create a chat community e.g. discord, slack etc.

**Reply by FaGru3n** [reply]
> create a chat community e.g. discord, slack etc.
chat commuinity already available within Eclipse Matrix Chat

**Comment by FaGru3n** [comment]
first steps were made an if we want to have a separate meeting slot we can generate a new one if we need one.
